package.json

```
@@ -7,7 +7,7 @@
     "format": "prettier --config integration/.prettierrc --write '*.{json,js}' 'integration/**/*.{js,ts}' '.github/**/*.{yml,yaml}'",
     "lint": "eslint --max-warnings=0 integration && prettier --config integration/.prettierrc --check '*.{json,js}' 'integration/**/*.{js,ts}'"
   },
-  "author": "WalletConnect, Inc. <walletconnect.com>",
+  "author": "Reown, Inc. <reown.com>",
```

Fixed branding: https://reown.com/terms-of-service

```suggestion
  "author": "reown inc. <reown.com>",
```